### ISSUE TITLE
Partition data storage of triples into per epoch

### ISSUE THREAD

**Issue by ChaoticTempest** [near/mpc#637]
We should partition our storage of triples into different partitions such that when we go to do resharing, the nodes themselves can still serve requests utilizing the old triples meanwhile we're doing the expensive resharing operation that could take hours. This will allow us to have no downtime

**Comment by volovyks** [comment]
Do you mean that we should keep triples for a different (old) participant set and use them while we are resharing AND generating a new set of triples?

**Comment by ChaoticTempest** [comment]
yup, it's to allow us to maintain two different MPC networks while resharing
